olad/Olad.cpp

```diff
@@ -171,6 +175,18 @@ int main(int argc, char *argv[]) {
       ola::NewCallback(olad->GetOlaServer(), &ola::OlaServer::ReloadPlugins));
 #endif  // _WIN32
 
+#if HAVE_LIBSYSTEMD
+  if (ola::SystemdNotifyAvailable()) {
+    OLA_INFO << "Systemd notification socket address present. "
```

Comment:
This probably wants to be a comma not a full stop (then adjust the case on the line below.